pkgs/development/python-modules/weasyprint/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib, isPy3k, fetchPypi, buildPythonPackage
+, cffi, pyphen, cairosvg, html5lib, tinycss2, cairocffi, cssselect2
+, pytestcov, pytestrunner, pytest-flake8, pytest-isort
+, glib, pango }:
+
+buildPythonPackage rec {
+  pname = "weasyprint";
+  version = "44";
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "WeasyPrint";
+    sha256 = "0f5bzwac1c4wgsscdv6d2d08n6g3j7pmj76ja5q31v40vzkz96vr";
+  };
+
+  buildInputs = [
+    pytestrunner
+  ];
+
+  propagatedBuildInputs = [
+    cffi
+    pyphen
+    cairosvg
+    html5lib
+    tinycss2
+    cairocffi
+    cssselect2
+  ];
+
+  checkInputs = [
+    pytestcov
+    pytest-isort
+    pytest-flake8
+  ];
+
+  preCheck = ''
+    export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:${lib.makeLibraryPath [ glib pango ]}"
```

Comment:
Is this only needed for the tests or to run the program in general?

Reply:
`glib` and `pango` are specifically needed for the tests to succeed, though looking at the external dependencies, `cairo` and `gdk_pixbuf` are needed for it to execute.